src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -89,20 +90,100 @@
     private static final long MB = 1024L * 1024;
     private static final long MINIMUM_FREE_SPACE = 200 * MB;
```

Comment:
I guess this should more wide be used, or thrown out 🤔 maybe even a combination of (a) minimum absolute free space (e.g., 200mb) and (b) not more than 95% of disk space used...